scripts/dev-dotnet.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, it was just a utility script I wrote while testing this PR :). It could be in it's own PR but I think that's probably overkill.